app/forms/SubscriptionsForm.scala

```diff
@@ -54,6 +55,8 @@ object SubscriptionsForm {
     "address" -> addressDataMapping
   )(PersonalData.apply)(PersonalData.unapply)
 
+  val productRatePlanIdMapping = mapping("ratePlanId" -> text)(ProductRatePlanId.apply)(ProductRatePlanId.unapply)
```

Comment:
so I hope this was the right thing to do here, I felt converting the string to a ProductRatePlanId should happen as early as possible